torch/__init__.py

```diff
@@ -210,16 +210,16 @@ def __init__(self, node):
         # class has a field named node that stores SymNode
         self.node = node
 
-    # Magic methods installed later
-
     def __bool__(self):
         return self.node.bool_()
 
     def __int__(self):
         return self.node.int_()
 
+    # Magic methods installed by torch.fx.experimental.symbolic_shapes
+
```

Comment:
Why leave an empty method here? Just for typing?

Reply:
typing yup